templates/admin.tpl.php

```diff
@@ -0,0 +1,127 @@
+<?php
+$module_path = backdrop_get_path('module', 'tawk_to');
+backdrop_add_css($module_path . '/css/tawk_to.admin.css', 'file');
+?>
+
+<div id="content" >
+  <div class="l-wrapper">
+    <div class="l-wrapper-inner container container-fluid">
+      <div class="l-messages success-message-container" role="status" aria-label="Status messages" id="optionsSuccessMessage">
+        <div class="messages status">
+          <h2 class="element-invisible">Status message</h2>
+          Successfully set widget options to your site
+        </div>
+      </div>
+      <div class="l-page-title">
+        <a id="main-content"></a>
+      </div>
+      <div class="l-content" role="main" aria-label="Main content">
```

Comment:
All classes that begin with `l-` are defined by (and added by) the layout system. Adding additional elements to the page that use these same classes is likely to cause problems.

The same goes for `role` and `aria-` attributes. These attributes are already on the page, on the appropriate elements. Duplicating them will cause issues for screen readers. 

**Are you sure including all these additional wrappers is necessary?** 
This template file is only responsible for what appears inside the "Main content" block on an admin page. The Messages, Page title, and everything outside that block will come from other sources. 

To change the page title, you can use `backdrop_set_title()`. To set a message, you can use `backdrop_set_message()`. Both of these functions can be placed in the page callback `tawk_to_admin_page` to set those elements on the admin page.

(If the additional wrappers are necessary, you should at least remove all the attributes that will cause problems for screen readers. And please test this page in a variety of different layouts to ensure that use of the `l-` layout classes doesn't cause larger issues.)